pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -133,13 +133,13 @@ var _ Manager = &manager{}
 
 // NewManager creates a new TopologyManager based on provided policy and scope
 func NewManager(topology []cadvisorapi.Node, topologyPolicyName string, topologyScopeName string, topologyPolicyOptions map[string]string) (Manager, error) {
-	klog.InfoS("Creating topology manager with policy per scope", "topologyPolicyName", topologyPolicyName, "topologyScopeName", topologyScopeName)
-
 	opts, err := NewPolicyOptions(topologyPolicyOptions)
 	if err != nil {
 		return nil, err
 	}
 
+	klog.InfoS("Creating topology manager with policy per scope", "topologyPolicyName", topologyPolicyName, "topologyScopeName", topologyScopeName, "topologyPolicyOptions", opts)
+
```

Comment:
why did we move the log?

Reply:
I wanted to print out `topologyPolicyOptions`, I can also leave old log and add another message when parsing opts is successful, WDYT?

Reply:
I think this is a good enough reasons for the move, we can keep it (provided there are no side effects. I think there are not, let's just doublecheck for peace of mind)

Reply:
The only change is that kubelet won't log policy and scope if `NewPolicyOptions` is failing.
Previously it would log them and fail afterwards.

Reply:
right, but the error is bubbled up. I think we can keep the change you made and move on.

Reply:
👍